pylint/config/option_manager_mixin.py

```diff
@@ -282,7 +282,7 @@ def read_config_file(self, config_file=None, verbose=None):
                 # Use this encoding in order to strip the BOM marker, if any.
                 with open(config_file, encoding="utf_8_sig") as fp:
                     parser.read_file(fp)
-                # normalize sections'title
+                # normalize section's title
```

Comment:
```suggestion
                # normalize each section's title
```

`_sections` is plural